src/Schema/Exception/TooManyValidSchemas.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\Schema\Exception;
+
+use cebe\openapi\spec\Schema;
+use Throwable;
+
+// Indicates that data was not matched against a combination of schema keywords
+class TooManyValidSchemas extends KeywordMismatch
+{
+    /** @var Schema[] */
+    protected $validSchemas = [];
+
+    /**
+     * @param mixed    $data
+     * @param Schema[] $validSchemas
+     *
+     * @return self
+     */
+    public static function fromKeywordWithValidSchemas(
+        string $keyword,
+        $data,
+        array $validSchemas,
+        ?string $message = null,
+        ?Throwable $prev = null
```

Comment:
Same question here for prev. It should no prev exceptions by design, I hope

Reply:
you're right; removed it.